cogs/commands/summarise.py

```diff
@@ -0,0 +1,90 @@
+import logging
+from typing import Optional
+
+import openai
+from discord import AllowedMentions
+from discord.ext import commands
+from discord.ext.commands import Bot, Context
+
+from cogs.commands.openaiadmin import is_author_banned_openai
+from config import CONFIG
+from utils.utils import split_into_messages
+
+LONG_HELP_TEXT = """
+Too much yapping? Summarise what people have said using the power of the GPT overlords!
+"""
+
+SHORT_HELP_TEXT = """Summarise messages."""
+
+mentions = AllowedMentions(everyone=False, users=False, roles=False, replied_user=True)
+chat_cmd = CONFIG.PREFIX + "chat"
+summarise_cmd = CONFIG.PREFIX + "summarise"
+
+
+def clean(msg, *prefixes):
+    for pre in prefixes:
+        msg = msg.strip().removeprefix(pre)
+    return msg.strip()
+
+
+class Summarise(commands.Cog):
+    def __init__(self, bot: Bot):
+        self.bot = bot
+        openai.api_key = CONFIG.OPENAI_API_KEY
+        self.model = "gpt-3.5-turbo"
```

Comment:
gpt4o-mini is cheaper and better (see chatgpt.py)
Also if you're only using 1 model move it to the top and constant rather than self

Reply:
also only ever use gpt4o-mini to summarise